src/applications/representative-appoint/tests/pages/representative/selectAccreditedRepresentative.unit.spec.jsx

```diff
@@ -408,7 +408,7 @@ describe('<SelectAccreditedRepresentative>', () => {
             await waitFor(() => {
               expect(searchBox).to.have.attr(
                 'error',
-                'Select the accredited representative or VSO you’d like to appoint below.',
```

Comment:
Similar thing here. You're changes shouldn't have changed how the no selection error appears.